pkg/roachprod/install/cluster_synced.go

```diff
@@ -1401,8 +1401,8 @@ func (c *SyncedCluster) Wait(ctx context.Context, l *logger.Logger) error {
 				}
 				return res, nil
 			}
-			res.Err = errors.New("timed out after 5m")
-			l.Printf("  %2d: %v", node, res.Err)
+			res.Err = errors.Wrapf(res.Err, "timed out after 5m")
```

Comment:
This change is worth making even if we never get to the bottom of the original issue David reported.

Reply:
True, thanks for the reminder. Changed the timeout back, and "%+v" -> "%v", so that only the wrap remains, which I think should be enough.